Kommander workspace edits and updates especially delete workspace was… (#3563)
mesosphere/dcos-docs-site: pages/dkp/kommander/1.4/workspaces/platform-service-dependencies/index.md

These platform services provide the functionality to collect logs over time from Kubernetes, platform services, and applications deployed on managed clusters. They also provide the ability to visualize and query the aggregated logs.

- [elasticsearch][elasticsearch]: A distributed, RESTful search and analytics engine

> - [elasticsearchexporter][elasticsearchexporter]: A Prometheus exporter for various metrics about Elasticsearch

Reviewer: ```suggestion
- [elasticsearchexporter][elasticsearchexporter]: A Prometheus exporter for various Elasticsearch metrics
```

> - [fluentbit][fluentbit]: A log processor and forwarder which allows you to collect any data like metrics and logs from different sources, enrich them with filters and send them to multiple destinations

Reviewer: ```suggestion
- [fluentbit][fluentbit]: A log processor that collects metrics and log data from different sources, enriches them with filters, and forwards them to multiple destinations.
```

These platform services provide monitoring capabilities by collecting metrics, including cost metrics, for Kubernetes, platform services, and applications deployed on managed clusters. They also provide the ability to visualize these metrics and evaluate rule expressions to trigger alerts.

> - [prometheus][prometheus]: A systems and service monitoring system that collects metrics from configured targets at given intervals, evaluates rule expressions, displays the results, and can trigger alerts when specified conditions are observed. **Note:** Prometheus Alertmanager and Grafana are included in the Prometheus bundled installation.

Reviewer: ```suggestion
- [prometheus][prometheus]: A systems and service monitoring system that collects metrics from configured targets at given intervals, evaluates rule expressions, displays the results, and triggers alerts when specified conditions are observed. **Note:** Prometheus Alertmanager and Grafana are included in the Prometheus bundled installation.
```

> - [prometheus-adapter][prometheus-adapter]: Exposes custom application metrics from Prometheus

Reviewer: ```suggestion
- [prometheus-adapter][prometheus-adapter]: Presents custom application metrics from Prometheus
```

Author: I think I want to keep exposes here in this case.

> - [kubecost][kubecost]: A cost-monitoring tool that gives you visibility into your Kubernetes resources to reduce spend and prevent resource-based outages

Reviewer: ```suggestion
- [kubecost][kubecost]: A cost-monitoring tool for Kubernetes resources that can reduce resources spent and prevent resource-based outages
```

Author: I think I want to keep it the other way, just because this phrasing is kind of sprinkled throughout the docs.
